### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [brainboxdotcc/dpp-comments#10] [General]
# index

A lightweight C++ Discord API library supporting the entire Discord API, including Slash Commands, Voice/Audio, Sharding, Clustering and more!

https://dpp.dev/index.html

**Comment by Codershell911** [comment]
It seens hard for me to understand and Write i beter stay with python

**Reply by ruslan-ilesik** [reply]
Yes, python can look simpler, and it's probably easier. But if your bot grows big, prepare your wallet for servers with a lot of ram and good cpu, while in c++ it would run on potato :)

**Reply by CyberDudeJ** [reply]
@ruslan-ilesik Honestly, all my C++ crap runs on a potato, it is wonderful.

**Comment by lunaradev1** [comment]
Can't wait to learn to use this, I'm learning C++ basics, so I hope this is worth it!

**Comment by S9kshi3** [comment]
its increadibly easy with c++ espicalyy using asio this version provides lowlevel efficent control which is hard to get in python and since it also works with unreal engine which i was searching for
